clouddriver-openstack/src/main/java/org/openstack4j/model/heat/builder/StackUpdateBuilder.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016 Target, Inc.
```

Comment:
I question our copyright on this, as it is mostly copy paste to add the stack updates